### DISCUSSION TITLE
Custom chimes removed after updates

### DISCUSSION THREAD

**Discussion by fabian19vs** [nimroddolev/chime_tts#44] [Q&A]
Is there a way to avoid losing the custom chimes uploaded to the mp3 directory and the setting changed in order to use the custom chimes, every time a new update comes up?

**Comment by nimroddolev** [comment]
Yes, you can use any file from your Home Assistant’s media folder and use the path to it in place of the options in the chime_path / end_chime_path dropdown lists

**Comment by nimroddolev** [comment]
Support has now been added to [set custom chimes in the drop-down chime list UI](https://github.com/nimroddolev/chime_tts/wiki/chime_tts.say#3-custom-chime-slots-for-the-chime-list-ui) which will persist after updating the integration
